app/src/main/java/ai/haptik/android/sample/app/Utils.java

```diff
@@ -2,27 +2,20 @@
 
 import ai.haptik.android.sdk.HaptikLib;
 import ai.haptik.android.sdk.InitData;
+import ai.haptik.android.sdk.picassohelper.PicassoApiFactory;
 import android.app.Application;
-import android.content.Context;
-import android.content.SharedPreferences;
 
 public class Utils {
-    static final String PREFS_KEY_HAPTIK_DATA_SYNC_ONCE = "prefs_key_haptik_data_sync_once";
-    static final String PREFERENCES_FILE_NAME = "HAPTIKLIB_DEMO_APP_PREFS";
-
-    static void setHaptikInitialDataSyncDone(Context context, boolean val) {
-        SharedPreferences sharedPreferences = context.getSharedPreferences(PREFERENCES_FILE_NAME, Context.MODE_PRIVATE);
-        sharedPreferences.edit().putBoolean(PREFS_KEY_HAPTIK_DATA_SYNC_ONCE, val).apply();
-    }
 
     public static InitData getHaptikInitData(Application application) {
         //Comment the setRunEnvironement for while releasing it to production
         HaptikLib.setRunEnvironment(HaptikLib.RUN_ENVIRONMENT_STAGING);
         return new InitData.Builder(application)
-            .baseUrl("ADD_BASE_URL_HERE")
+            .baseUrl("https://staging.hellohaptik.com")
```

Comment:
Can we remove this?

Reply:
Good catch 💃

Reply:
I think comments are insults enough, lets not appreciate them and rub salt on wounds 😝 😢 😅